pkg/controller/postgresql.go

```diff
@@ -40,12 +40,27 @@ func (c *Controller) clusterResync(stopCh <-chan struct{}, wg *sync.WaitGroup) {
 
 // clusterListFunc obtains a list of all PostgreSQL clusters
 func (c *Controller) listClusters(options metav1.ListOptions) (*acidv1.PostgresqlList, error) {
+	var pgList acidv1.PostgresqlList
+
 	// TODO: use the SharedInformer cache instead of quering Kubernetes API directly.
 	list, err := c.KubeClient.AcidV1ClientSet.AcidV1().Postgresqls(c.opConfig.WatchedNamespace).List(options)
 	if err != nil {
 		c.logger.Errorf("could not list postgresql objects: %v", err)
 	}
-	return list, err
+	if c.controllerID != "" {
+		c.logger.Debugf("watch only clusters with controllerID %q", c.controllerID)
+	}
```

Comment:
1. the `else` branch of this condition - i.e. watching all PG resources without explicit annotations  - is also worth a logging message just in case someone does  a misconfiguration
2. why is this message logged here and not at the controller (operator pod) start up ?

Reply:
1. The `else` part reflects the current behavior people are used to. I'm not sure if adding another log line would raise more confusion or trigger users to add annotations to their manifests.
2. Where you think is a good place in the controller? Maybe it's good to log it at each iteration of listClusters to remind people that this operator only listens to certain clusters.